packages/mui-material/src/Breadcrumbs/Breadcrumbs.js

```diff
@@ -152,7 +153,7 @@ const Breadcrumbs = React.forwardRef(function Breadcrumbs(inProps, ref) {
         key="ellipsis"
         slots={{ CollapsedIcon: slots.CollapsedIcon }}
         slotProps={{ collapsedIcon: collapsedIconSlotProps }}
-        onClick={handleClickExpand}
+        onClick={collapsedIconSlotProps.onClick || handleClickExpand}
```

Comment:
I think this should be either here or above on line 116 through `additionalProps`.

Reply:
I'm keeping this line and removing the line on 116 instead. Thanks